daml-model/daml/Model/LoanConfig.daml

```diff
@@ -36,18 +39,20 @@ template LoanConfig
 
     where
         signatory provider
-        observer accountHolder
+        observer getObserver accountHolder
         key (BalanceKey with iban = account, provider = provider) : BalanceKey
         maintainer key.provider
 
         ensure activeLoan >= 0.0
             && activeLoan <= maxLoanSize
             && balanceLoanThreshold >= 0.0
-            && balanceLoanThreshold <= maxLoanSize
```

Comment:
maxLoanSize is independent of the threshold.
i.e. you could say "I'm willing to give you a loan up to 1000 USD, and you can top up your account for any value below 1'000'000 .. i.e. if you get to 999'500, you can get a 500USD loan to bump it up to a million again."

Reply:
Agreed